src/index.ts

```diff
@@ -62,18 +63,17 @@ export type BentoSettings = {
     createdAt?: string;
   } & DynamicAttributes;
   /**
-   * Account users represent employees at your account.
-   * For example, someone named John Doe who works at AcmeCo.
-   * Or the person who would be using your software at the company who purchased it.
+   * Account users represent people at your customer-companies.
+   * For example, someone named John Doe who works at AcmeCo and uses your product.
    */
   accountUser: {
     /**
      * Unique identifier for the account user
-     * This is best set to something used internally by your app
+     * This is best set to something used internally by your app like an id.
      */
     id: string;
     /**
-     * (RECOMMENDED) Human-readable unique account user identifier
+     * (Recommended, but not required) Human-readable unique account user identifier
```

Comment:
nit: Whether or not required is specified by the types themselves, so I considered it redundant to mention it in the description as well.

In this case, the `?` right after `fullName` indicates to Typescript that this property is optional

Reply:
cool, I'll leave it to you. Decided to clarify bc we get asked so many questions about this.